scripts/apothecary/formulas/openssl/openssl.sh

```diff
@@ -652,6 +652,7 @@ function copy() {
 		git checkout $1/include/openssl/opensslconf_osx.h
     	git checkout $1/include/openssl/opensslconf_android.h
     	git checkout $1/include/openssl/opensslconf_win32.h
+    	git checkout $1/include/openssl/opensslconf_vs.h
```

Comment:
Ah ok, I was using the of master and it kept complaining that conf_vs was missing, makes more sense now